controllers/movies.js

```diff
@@ -28,4 +28,23 @@ const getMovieByTitle = (request, response) => {
   return response.send(foundMovie)
 }
 
-module.exports = { getAllMovies, getMovieByTitle, checkInput }
+const saveNewMovie = (request, response) => {
+  const {
+    title, directors, releaseDate, rating, runTime, genres
+  } = request.body
+
+  if (!title || !directors || !releaseDate || !rating || !runTime || !genres) {
+    // eslint-disable-next-line max-len
+    return response.status(400).send('The following fields are required: title, directors, releaseDate, rating, runTime, genres')
+  }
```

Comment:
We would only ever disable max-len for a test file or data files, never code. This should be multi-lined.

```JavaScript
return response
  .status(400)
  .send('The following fields are required: title, directors, releaseDate, rating, runTime, genres')

```